contrib/download-frozen-image-v2.sh

```diff
@@ -343,6 +344,7 @@ while [ $# -gt 0 ]; do
 									-H 'Accept: application/vnd.docker.distribution.manifest.v2+json' \
 									-H 'Accept: application/vnd.docker.distribution.manifest.list.v2+json' \
 									-H 'Accept: application/vnd.docker.distribution.manifest.v1+json' \
+									-H 'Accept: application/vnd.oci.image.manifest.v1+json' \
```

Comment:
I think we should swap the order (OCI first, so we're still in descending order of preference) and include the OCI image index type too: :eyes:

```suggestion
									-H 'Accept: application/vnd.oci.image.manifest.v1+json' \
									-H 'Accept: application/vnd.oci.image.index.v1+json' \
									-H 'Accept: application/vnd.docker.distribution.manifest.v2+json' \
									-H 'Accept: application/vnd.docker.distribution.manifest.list.v2+json' \
									-H 'Accept: application/vnd.docker.distribution.manifest.v1+json' \
									-H 'Accept: application/vnd.oci.image.manifest.v1+json' \
```